modules/claims_api/app/services/claims_api/evidence_waiver_document_service.rb

```diff
@@ -7,13 +7,24 @@ class EvidenceWaiverDocumentService < DocumentServiceBase
 
     def create_upload(claim:, pdf_path:, doc_type:, ptcpnt_vet_id:)
       unless File.exist? pdf_path
-        ClaimsApi::Logger.log(LOG_TAG, detail: "Error creating upload doc: #{pdf_path} doesn't exist,
-                                                    claim_id: #{claim.claim_id}")
+        ClaimsApi::Logger.log(LOG_TAG, ews_id: claim.id, claim_id: claim.claim_id,
+                                       detail: "Error creating upload doc: #{pdf_path} doesn't exist.")
         raise Errno::ENOENT, pdf_path
       end
 
+      ClaimsApi::Logger.log(LOG_TAG, ews_id: claim.id, claim_id: claim.claim_id,
```

Comment:
Instead of logging twice for every successful upload (once here and the other on the success) - let's have the success message log out the duration. That way we'll do half as much logging but can still see how long the upload process took.